sklearn/utils/estimator_checks.py

```diff
@@ -483,8 +484,61 @@ def check_dict_unchanged(name, Estimator):
                               'Estimator changes __dict__ during %s' % method)
 
 
+def _is_public(attr):
```

Comment:
this name is not really appropriate. maybe it should be ``is_init_parameter`` and should return the inverse?